src/lib/mina.ts

```diff
@@ -429,8 +429,8 @@ function LocalBlockchain({
     getNetworkId: () => minaNetworkId,
     proofsEnabled,
     /**
-   * @deprecated use {@link Mina.getNetworkConstants}
-   */
+     * @deprecated use {@link Mina.getNetworkConstants}
+     */
```

Comment:
@shimkiv you should set up `prettier` :P

Reply:
Done.